src/main/java/tr/edu/ogu/ceng/service/StudentService.java

```diff
@@ -3,7 +3,7 @@
 
 import org.springframework.stereotype.Service;
 import javax.persistence.EntityNotFoundException;
-import org.springframework.beans.factory.annotation.Autowired;
+//import org.springframework.beans.factory.annotation.Autowired;
```

Comment:
neden comment?

Reply:
Kullanılmadığı için yorum satırına almıştım hocam.